src/logic/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -1,29 +1,37 @@
 import { ThunkDispatch } from 'redux-thunk'
 import { AnyAction } from 'redux'
+import { parse } from 'querystring'
```

Comment:
Can we use the native URL parser instead?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL – it has a `searchParams` property

Reply:
I'll pivot to using `URLSearchParams` as I did in `getHistoryTxListPage` instead.